src/NodaTime.Web/Markdown/1.0.x/concepts.md

```diff
@@ -144,18 +144,20 @@ change occurs - usually for daylight saving changes.
 Most of the time when you use a `DateTimeZone` you won't need
 worry about that - the main purpose is usually to convert between a
 [`ZonedDateTime`][ZonedDateTime] and a [`LocalDateTime`][LocalDateTime]
-- where the names mean exactly what you expect them to. There's a slight
+, where the names mean exactly what you expect them to. There's a slight
```

Comment:
Nit: could you put the comma at the end of the previous line instead of the start of this one?
I'm surprised it's being treated as a list start anyway without an extra line break, but using a comma is a perfectly good fix.